src/Board/index.jsx

```diff
@@ -0,0 +1,215 @@
+        setGameStatus(GameStatus.gameOver);
+        setH1Title(H1Title.playerWon);
+        showWinner();
+      }
+    }
+  };
+
+  const checkIfDraw = () => {
+    let allResultsArray = [];
+    for (const row of Object.values(board)) {
+      allResultsArray.push(row);
+    }
+    allResultsArray = allResultsArray.flat();
+    allResultsArray = allResultsArray.filter((result) => result.value === null);
+    if (allResultsArray.length === 0) {
+      setGameStatus(GameStatus.draw);
+      setH1Title(H1Title.draw);
+      showDraw();
+    }
+  };
+
+  const getFieldById = (fieldIdToFind) => {
+    for (const rowKey in board) {
+      const rowArray = board[rowKey];
+      const foundField = rowArray.find((field) => field.id === fieldIdToFind);
+      if (foundField) {
+        return foundField;
+      }
+    }
+  };
+
+  const move = (fieldId) => {
+    const field = getFieldById(fieldId);
+    if (!field.disabled && turn === Turn.playerOne) {
+      field.value = turn;
+      setBoard({ ...board });
+      setTurn(Turn.playerTwo);
+      setArrowToSecondPlayer();
+      checkIfWonOrDrawAndSetGameStatus();
+    } else if (!field.disabled && turn === Turn.playerTwo) {
+      field.value = turn;
+      setBoard({ ...board });
+      setTurn(Turn.playerOne);
+      setArrowToFirstPlayer();
+      checkIfWonOrDrawAndSetGameStatus();
+    }
+  };
+
+  const setArrowToFirstPlayer = () => {
+    if (gameStatus === GameStatus.inProgress) {
+      setArrowFirstPlayerClass(styles.arrowForPlayerOn);
+      setArrowSecondPlayerClass(styles.arrowForPlayerOff);
+    }
+  };
+
+  const setArrowToSecondPlayer = () => {
+    if (gameStatus === GameStatus.inProgress) {
+      setArrowFirstPlayerClass(styles.arrowForPlayerOff);
+      setArrowSecondPlayerClass(styles.arrowForPlayerOn);
+    }
+  };
+
+  const makeMoveAndChangeTurn = (fieldId) => {
+    if (gameStatus === GameStatus.inProgress) {
+      move(fieldId);
+    } else if (gameStatus === GameStatus.draw) {
+      showDraw();
+    } else {
+      showWinner();
+    }
+  };
+  const showDraw = () => {
+    setArrowSecondPlayerClass(styles.arrowForPlayerOff);
+    setArrowFirstPlayerClass(styles.arrowForPlayerOff);
+  };
+  const showWinner = () => {
+    if (turn === Turn.playerOne) {
+      setArrowToFirstPlayer();
+    } else {
+      setArrowToSecondPlayer();
+    }
+  };
+
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.board}>
+        {Object.keys(board).map((singleRowKey) => (
+          <Row
+            key={singleRowKey}
+            rowOfFields={board[singleRowKey]}
+            makeMoveAndChangeTurn={makeMoveAndChangeTurn}
+            turn={turn}
+            gameStatus={gameStatus}
+          />
+        ))}
+      </div>
+      <div className={styles.infoBox}>
+        <h1>{h1Title}</h1>
+        <div className={styles.arrowAndPlayerContainer}>
+          <BsArrowRightShort className={arrowFirstPlayerClass} />
```

Comment:
```suggestion
          <BsArrowRightShort className={turn === Turn.playerOne ? styles.arrowForPlayerOn : null} />
```
This way you can get rid of 2 (`arrowFirstPlayerClass`, `arrowSecondPlayerClass`) state variables

Reply:
done